### ISSUE TITLE
Buggy Slash Commands menu

### ISSUE THREAD

**Issue by SomeAspy** [GooseMod/GooseMod#61]
**Describe the bug**
The bug causes the slash commands menu that pops up when you enter slash to fire errors in console, and will send you to the top of the list.
Tested in a new discord server, on a clean install of Windows 11 in a VM.
This does not happen without goosemod injected.

**To Reproduce**
1. Enter / into the message box
2. Attempt to scroll to bottom of list
3. Observe bug.

**Expected behavior**
The list of slash commands should scroll smoothly without an issue.

**Screenshots**
I have a video of attempting to reproduce the bug with and without goosemod injected. they will not upload. if you need them let me know.

**GooseMod info:**
GooseMod 12.3 (14fbf22)
GooseMod Ext 1.2.0
Clean install of goosemod

**Discord info:**
 - Update Channel: Development
 - OS: Windows 11 Clean install
 - Host Version: 1.0.640
 - Web Version:  Canary 111058 (5382b66)

**Comment by Toasty9004Dev** [comment]
I got the Bug too
**Describe the bug**
The bug causes the slash commands menu that pops up when you enter slash to fire errors in console, and will send you to the top of the list.
I can't touch any bots icon
This does not happen without GooseMod injected.

**To Reproduce**
Steps to reproduce the behavior:

1. Enter / into the message box
2. Attempt to scroll to bottom of list
3. Observe bug.

**Expected Behavior**
The list of slash commands sometimes should scroll smoothly without an issue and sometimes the lagged up to the top of the list.

**Screenshot**

https://user-images.githubusercontent.com/90614108/150662200-9f7c068a-844e-4b34-9784-fdcba70b5a75.mp4

**GooseMod info**
GooseMod 12.3 (14fbf22)
GooseMod Ext 1.2.1

**Discord info**

- Update Channel: Stable
- OS (PC 1): Windows 10 64-bit (10.0.19043.928) 
- OS (PC 2): Windows 8.1 (6.3.9600)
(I tested on both of my computer)
- Web Version: Stable 111699 (e8f9fb9)
